src/fragments/lib/logging/common/remote_configuration/remote_configuration.mdx

```diff
@@ -0,0 +1,230 @@
+You can remotely configure the Amplify Logger, enabling you to make changes to your logging levels or user allow list in your deployed applications.
+
+<Callout>
+The logging configurations you set remotely will overwrite the local log level and persist for future app sessions.
+</Callout>
+
+## Set up remote configuration backend resources
+
+<Callout>
+
+You can use Amplify CLI generate a CDK template and [add custom resources](https://docs.amplify.aws/cli/custom/cdk/).  
+
+</Callout>
+
+Below is an example CDK construct that provisions the Amazon CloudWatch, AWS API Gateway, AWS Lambda, and AWS S3 bucket.  The CDK construct also deploys the `remoteloggingconstraints.json` file to the AWS S3 bucket. 
+
+### Permissions
+The CDK construct creates IAM policies and assigned them to the Amplify authenticated and unauthenticated roles. 
+1. The API Gateway endpoint can only be invoked by user with the Amplify authenticated or unauthenticated roles. 
+2. The AWSCloudWatch logs can only be created and sent by users with the Amplify authenticated or unauthenticated roles.
+3. The lambda invocation is only allowed from the provisioned AWS API Gateway resource.
+4. The S3 bucket containing the `remoteloggingconstraints.json` can only be read by the Lambda execution role.
+
+### Replace the placeholder values with your own values:
+* `<CloudWatch-log-group-name>` is the log group that logs will be sent to.  Note that this CDK construct sample includes logic to create the CloudWatch log group that you may have are created in previous steps to [setup backend resources](/lib/logging/setup-logging/#provision-backend-resources).
```

Comment:
create the CloudWatch log group that you may have already created in previous steps.

Reply:
Text updated.